### PR TITLE
feat: set up default master locale

### PR DESCRIPTION
**Pull request by BohdanOne**
**Resolves**: [DT-2482](https://linear.app/prismic/issue/DT-2482/set-default-master-language-when-creating-new-repo-via-cli)

### Description

When a user creates a repository through the terminal (without starter), it should set a default master lang (en-us) for the repository.

### Checklist

- [ ] A comprehensive Linear ticket, providing sufficient context and details to facilitate the review of the PR, is linked to the PR.
- [ ] If my changes require tests, I added them.
- [ ] If my changes affect backward compatibility, it has been discussed.
- [ ] If my changes require an update to the CONTRIBUTING.md guide, I updated it.

### How to QA [^1]

[^1]:
	Please use these labels when submitting a review:
	:question: #ask:&ensp;Ask a question.
	:bulb: #idea:&ensp;Suggest an idea.
	:warning: #issue:&ensp;Strongly suggest a change.
	:tada: #nice:&ensp;Share a compliment.

### CODE REVIEW

File: packages/init/src/SliceMachineInitProcess.ts [hunk 1]
```diff
@@ -184,6 +184,9 @@ export class SliceMachineInitProcess {
 			}
 			if (!this.context.repository.exists) {
 				await this.createNewRepository();
+				if (!this.options.starter) {
```
Comment (xrutayisire) [inline packages/init/src/SliceMachineInitProcess.ts]: ⚠️ Technically, this is wrong, and we can and will still have the bug. You assume here you start with a starter with the command the UI gave you, but the bug happens actually when you don't start with the UI. 

Imagine you start for here: https://github.com/prismicio-community/nextjs-starter-prismic-minimal/tree/master
Or from here: https://prismic.io/docs/nextjs-example-projects

As you see, there are no starter option. The condition to check if you need to setup the master locale is if there are documents to push. If you have documents to push, it's what we consider a starter today.

File: packages/init/src/SliceMachineInitProcess.ts [hunk 2]
```diff
@@ -965,6 +968,20 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 		]);
 	}
 
+	protected setDefaultMasterLocale(): Promise<void> {
+		return listrRun([
+			{
+				title: `Setting default master language...`,
```
Comment (xrutayisire) [inline packages/init/src/SliceMachineInitProcess.ts]: 💡 I don't think we are talking about `master` language in the product, no?

See the title of the PB onboarding step: `What's your main content language?` 

I would do something like:

```suggestion
				title: `Setting main content language...`,
```

And also:

```
task.title = `Main content language set to ${chalk.cyan(
  "English - United States",
)} 🇺🇸. You can change it anytime in your project settings.`;
```

I would also use the label, as in the select in PB.

**Review by xrutayisire — CHANGES_REQUESTED** [review]

File: packages/init/src/SliceMachineInitProcess.ts (hunk 2 above)
Comment (BohdanOne) [reply packages/init/src/SliceMachineInitProcess.ts]: Ok, will confirm the change of en-Us to the label with Come

File: packages/init/src/SliceMachineInitProcess.ts (hunk 1 above)
Comment (BohdanOne) [reply packages/init/src/SliceMachineInitProcess.ts]: Ok, thanks for pointing that out!
Changed

File: packages/init/src/SliceMachineInitProcess.ts (hunk 2 above)
Comment (BohdanOne) [reply packages/init/src/SliceMachineInitProcess.ts]: message change confirmed with Come, updated

File: packages/init/src/SliceMachineInitProcess.ts (hunk 1 above)
Comment (xrutayisire) [reply packages/init/src/SliceMachineInitProcess.ts]: Hum so you decide to not check if there are documents to push?

Comment (BohdanOne) [reply packages/init/src/SliceMachineInitProcess.ts]: ah, ok. I misunderstood your suggestion. Going back to this.

**Review by xrutayisire — APPROVED** [review]
LGTM!

**Pull request merged**